content/en/getting_started/devsecops/_index.md

```diff
@@ -7,7 +7,7 @@ This guide introduces the DevSecOps bundles with links to setup instructions to
 
 ## APM DevSecOps
 
-The APM DevSecOps bundles combine [Application Performance Monitoring (APM)][4] with the [Application Vulnerability Management][1] capabilities of [Application Security Management (ASM)][2].
+The APM DevSecOps bundles combine [Application Performance Monitoring (APM)][4] with the [Application Vulnerability Management][3] capabilities of [Application Security Management (ASM)][2].
```

Comment:
```suggestion
The APM DevSecOps bundles combine [Application Performance Monitoring (APM)][4] with the [Application Vulnerability Management][10] capabilities of [Application Security Management (ASM)][2].
```
link [3] goes to the CSM docs actually, what link did you want this to be here for AVM ? I am thinking it should be 10 as this is the getting started with AVM page?